packages/chakra-ui/src/components/ui/slider.tsx

```diff
@@ -0,0 +1,80 @@
+import { Slider as ChakraSlider, HStack } from '@chakra-ui/react';
+import * as React from 'react';
```

Comment:
Two things. First, put the react imports up top. Secondly, don't use the `import * as React from 'react';` pattern. Instead import only that which you need. Like:
`import { ReactNode, forwardRef } from 'react'`;